examples/basic/chat-search.py

```diff
@@ -129,6 +129,9 @@ def chat(opts: CLIOptions) -> None:
 def main(
     debug: bool = typer.Option(False, "--debug", "-d", help="debug mode"),
     model: str = typer.Option("", "--model", "-m", help="model name"),
+    provider: str = typer.Option(
+        "sciphi", "--provider", "-m", help="search provider name (Google, SciPhi)"
```

Comment:
`-p`? otherwise, it's gonna be the same as the model flag

Reply:
mistake, sorry.